### DISCUSSION TITLE
[Вопрос] дистанция и агро ботов

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#230] [Q&A]
Всем привет!!!
У меня вопрос на примере:
Играю сейчас элемшаманом. Раньше за рдд особо не играл. В конфигнастройках NpcBot.BaseFollowDistance по умолчанию 30.  Вообщем столкнулся с проблей и нужна помощь в настройке. Дистанция атаки у шамана 40м, поэтому танки не агрятся и рдд и милидд не атакуют при первой моей атаке. 
trickerer, помоги настроить на примере чтоб при  ренджатаке моего шамана элема все боты стартовали атаковать. Мануал твой читал через переводчик, но есть вопрос поэтому и задаю его.

**Comment by trickerer** [comment]
`NpcBot.BaseFollowDistance` это расстоние от игрока, дальше которого боты не отдаляются для атаки. Поставишь 45 - и боты с гарантией сразу будут атаковать.
В данжах высокие значения этого параметра не всегда оптимальны. Можно конечно использовать приказ ждать на месте, но часто проще поставить дистанцию следования в 10 (например), и агрить из-за угла. `.npcb dist 10`
